gen/tests/test_config.py

```diff
@@ -815,6 +815,7 @@ def test_exhibitor_admin_password_obscured():
     assert yaml.load(generated.arguments['config_yaml_full'])[var_name] == var_value
 
 
+@pytest.mark.skipif(pkgpanda.util.is_windows, reason='TODO: Needs porting on Windows')
```

Comment:
Hi @sschneid  - Can we add a JIRA against TODO. We have a recommendation on how to add TODO comment here - https://github.com/dcos/dcos/blob/master/docs/styleguides/python.md